testing/code/test_excinfo.py

```diff
@@ -925,3 +925,14 @@ def test_repr_traceback_with_unicode(style, encoding):
     repr_traceback = formatter.repr_traceback(e_info)
     assert repr_traceback is not None
 
+
+@pytest.mark.xfail
+def test_cwd_deleted_(testdir):
```

Comment:
Sorry for not being clearer, I meant something like this:

```python
def test_cwd_deleted(testdir):
    testdir.makepyfile("""
        def test(tmpdir):
            tmpdir.chdir()
            tmpdir.remove()
            assert False         
    """)
    result = testdir.runpytest() 
    result.stdout.fnmatch_lines(['* 1 failed in *'])
    assert 'INTERNALERROR' not in result.stdout.str() + result.stderr.str()
```

This way we make sure that when the test passes we are observing the correct behavior.